app/assets/javascripts/application.js

```diff
@@ -12,5 +12,12 @@
 //
 //= require jquery
 //= require jquery_ujs
+//= require masonry/jquery.masonry
 //= require turbolinks
 //= require_tree .
+$(function(){  
+  $('#brick').masonry({
+    itemSelector: '.img',
+    isFitWidth: true
+  });
+});  
```

Comment:
Prefer jQuery's more verbose syntax here as the `$(...)` shorthand is obscure. This code is clearer:

```javascript
$(document).ready(function(){  
  $('#brick').masonry({
    itemSelector: '.img',
    isFitWidth: true
  });
});
```
http://learn.jquery.com/using-jquery-core/document-ready/